openlibrary/plugins/openlibrary/js/page-barcodescanner/index.js

```diff
@@ -69,3 +77,15 @@ export function init() {
 function isBarcodeISBN(code) {
     return code.startsWith('97');
 }
+
+function CheckISBN(code, confirmation_list) {
```

Comment:
```suggestion
function checkISBN(code, confirmation_list) {
```
The uppercase `C` makes it seem like this is a class, instead of a function.

Reply:
It would also be good to include a JSDoc comment documenting what this function does.  This will make things easier for future contributors and maintainers.